mappings/net/minecraft/world/gen/trunk/TrunkPlacer.mapping

```diff
@@ -0,0 +1,27 @@
+CLASS net/minecraft/class_5141 net/minecraft/world/gen/trunk/TrunkPlacer
+	FIELD field_23759 type Lnet/minecraft/class_5142;
+	FIELD field_23760 baseHeight I
+	FIELD field_23761 firstRandomHeight I
+	FIELD field_23762 secondRandomHeight I
+	METHOD <init> (IIILnet/minecraft/class_5142;)V
+		ARG 1 baseHeight
+		ARG 2 firstRandomHeight
+		ARG 3 secondRandomHeight
+		ARG 4 type
+	METHOD method_26990 getBaseHeight ()I
+	METHOD method_26991 generate (Lnet/minecraft/class_3747;Ljava/util/Random;ILnet/minecraft/class_2338;ILjava/util/Set;Lnet/minecraft/class_3341;Lnet/minecraft/class_4640;)Ljava/util/Map;
+		COMMENT Generates the trunk blocks and return a map from branch tops to the
+		COMMENT radius allowed for foliage generation on that branch top.
+		ARG 1 world
+		ARG 2 random
+		ARG 3 trunkHeight
+		ARG 4 pos
+		ARG 5 foliageRadius
+		ARG 6 logs
+		ARG 7 box
+		ARG 8 config
+	METHOD method_26992 serialize (Lcom/mojang/datafixers/types/DynamicOps;)Ljava/lang/Object;
+		ARG 1 ops
+	METHOD method_26993 getHeight (Ljava/util/Random;Lnet/minecraft/class_4640;)I
```

Comment:
I assume this method returns the height after the random was applied, so maybe `getFinalHeight` would be better?

Reply:
the height field should probably be renamed to base height than editing this method

Reply:
However the `height` and `height_random` are fields from mojang, I can still rename `height` to `baseHeight`, `offset` to `baseOffset` if you'd prefer. @Runemoro how's your opinion?

Reply:
I think it's clear `getHeight` is the final height. Any other height would be named something like `getBaseHeight`.

Reply:
There is a field in this class called `height` (as seen in data structure), but the field is effectively a base height. Should that field be renamed from `height` to `baseHeight`

Reply:
Oh. If it matches the Mojang name then I think it's ok.